src/fury/meta/preprocessor.h

```diff
@@ -25,8 +25,15 @@
 // passed to the macros are fully expanded before they are concatenated.
 #define FURY_PP_CONCAT(a, b) FURY_PP_CONCAT_IMPL(a, b)
 
+#ifdef _WIN32
+#define EXTEND(...) __VA_ARGS__
+#define FURY_PP_NARG_IMPL(...) EXTEND(FURY_PP_NARG_CALC(__VA_ARGS__))
+#define FURY_PP_NARG(...)                                                      \
+  EXTEND(FURY_PP_NARG_IMPL(__VA_ARGS__, FURY_PP_NARG_REV()))
+#else
```

Comment:
Could you explain these changes?

Reply:
Because msvc will not expand the macro and will only pass it in as a parameter, this will lead to many syntax errors.